sdk/core/azure-core/src/main/java/com/azure/core/implementation/http/policy/InstrumentationPolicy.java

```diff
@@ -18,16 +18,21 @@
 import com.azure.core.util.tracing.Tracer;
 import reactor.core.publisher.Mono;
 
+import java.net.URL;
+
+import static com.azure.core.http.policy.HttpLoggingPolicy.RETRY_COUNT_CONTEXT;
 import static com.azure.core.util.tracing.Tracer.DISABLE_TRACING_KEY;
 
 public class InstrumentationPolicy implements HttpPipelinePolicy {
-    private static final String HTTP_USER_AGENT = "http.user_agent";
-    private static final String HTTP_METHOD = "http.method";
-    private static final String HTTP_URL = "http.url";
-    private static final String HTTP_STATUS_CODE = "http.status_code";
+    private static final String HTTP_REQUEST_METHOD = "http.request.method";
+    private static final String HTTP_RESEND_COUNT = "http.request.resend_count";
+    private static final String URL_FULL = "url.full";
+    private static final String SERVER_ADDRESS = "server.address";
+    private static final String SERVER_PORT = "server.port";
+    private static final String HTTP_RESPONSE_STATUS_CODE = "http.response.status_code";
```

Comment:
I hadn't noticed this before that OpenTelemetry semconv is natively baked into `azure-core`

Reply:
they are! (we still can remap them to something else in the plugin though)